### PR TITLE
contour: add version 0.3.3.204

### PR DESCRIPTION
**Pull request by SonaliBendre**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/contour.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/contour-terminal/contour/releases/download/v0.3.3.204/contour-0.3.3.204-win64.zip",
+            "hash": "819a822afbd91a5d541ef94e05fe1f886616d724ead87c3bd508c67cfe70941f",
+            "extract_dir": "Contour-0.3.3.204-win64/bin"
+        }
+    },
+    "bin": "contour.exe",
+    "extract_dir": "",
```
Comment (issaclin32) [inline bucket/contour.json:14]:
```suggestion
```

File: bucket/contour.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "shortcuts": [
+        [
+            "contour.exe",
+            "contour"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/contour/contour-terminal/releases"
+    },
```
Comment (issaclin32) [inline bucket/contour.json:23]:
```suggestion
    "checkver": "github",
```

**Comment by issaclin32** [comment]
config location: `$Env:LocalAppData\contour`
(This is a reminder. No need to modify anything regard to this.)

**Comment by SonaliBendre** [comment]
@issaclin32 I think it can be merged now

**Pull request merged**